packages/apps/contracts/HighRollerApp.sol

```diff
@@ -33,7 +34,6 @@ contract HighRollerApp is CounterfactualApp {
   }
 
   struct AppState {
-    address[2] playerAddrs;
```

Comment:
This was done in another PR, I think. Can you rebase on `master`?

Reply:
It's not done in master